vue/src/components/Chart.vue

```diff
@@ -129,9 +129,11 @@ export default {
     // functionの中からは直接storeに保存できない？
     // dataの中では定義できないので、ここで定義する？
     setComment () {
-      const comment = []
-      this.checkContents.map((Comment) => {
-        comment.push(Comment.comment)
+      const comments = []
+      this.checkContents.map((contents) => {
+        // 用意したcommentsという箱に、contents.ageという配列を持ったageと、contents.commentという配列を持った
+        // commentをObjectとしてpushする
+        comments.push({ age: contents.age, comment: contents.comment })
       })
```

Comment:
```suggestion
      const comments = this.checkContents.map((content) => {
        return { age: content.age, comment: content.comment }
      })
```
こんな感じにすると、よりシンプルに記載できます。

Reply:
遅くなりました！対応しました！